### DISCUSSION TITLE
Blog: Cucumber Testing

### DISCUSSION THREAD

**Discussion by open-schnick** [FileFighter/filefighter.github.io#21] [General]
Leave your comments here about our Cucumber Testing Blog :D
Your FileFighter Team

**Comment by ThSilv3r** [comment]
Hello FileFighter Team

Your .feature files are looking very good. They are structured and seem to have every scenario in them which exists.
Also the description how you use Cucumber in your blog gives tips how to use it  efficienty.

We look forward to hearing from you.

Your Team SweatForSuccess

**Comment by Mueller-Patrick** [comment]
Dear FileFighter Team!

I think you did a solid job writing the cucumber tests. You included positive as well as negative tests, you prevent duplicate code and you cover a lot of scenarios. That's a pretty good base to start developing from. The only small thing I noticed is that you have the package ...cucumber/de/filefighter/rest for the .feature files and ...java/de/filefighter/rest/cucumber for the stepdefs files, which is a bit weird in my opinion, but I'm not that familiar with your overall project structure so it might as well be that this just makes sense for your structure.
Also, one thing I specifically liked about your blog post is your "Blogging tip of the day" which was really helpful for me, so thank you a lot for including this :)

I really look forward to reading your next blog posts.

Patrick from [Betterzon](https://blog.betterzon.xyz)

**Comment by open-schnick** [comment]
Thanks for your kind feedback @Mueller-Patrick and @ThSilv3r :D
